### PR TITLE
MAML changes to fix ignite

### PR DESCRIPTION
**Pull request by avijitgupta**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@avijitgupta__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (avgupt). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist.

File: src/ServiceManagement/Network/Network.Tests/Utilities.cs [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                imagesList.Images.FirstOrDefault(i => string.Equals(i.OperatingSystemType, "Windows", StringComparison.OrdinalIgnoreCase));
+
+            VirtualMachineOSImageGetResponse gottenImage = computeClient.VirtualMachineOSImages.Get(imageToGet.Name);
+
+            VirtualMachineCreateDeploymentParameters parameters = CreateVMParameters(gottenImage, deploymentName, storageAccountName, "SampleLabel", blobUrl);
+
+            parameters.Roles[0].ConfigurationSets.Add(new ConfigurationSet
+            {
+                AdminUserName = "testuser",
+                AdminPassword = "@zur3R0ck5",
```
Comment (hovsepm) [inline src/ServiceManagement/Network/Network.Tests/Utilities.cs]: What is this user name and password? Are they used against any existing configuration?

Comment (avijitgupta) [inline src/ServiceManagement/Network/Network.Tests/Utilities.cs]: They are just random username and password used for creating a VM.

**Pull request merged**